app/src/main/res/values/strings.xml

```diff
@@ -480,4 +481,5 @@
   <string name="developer_options_force_crash">Force crash</string>
   <string name="all">All</string>
   <string name="mark_completed_uppercase">MARK COMPLETED</string>
+  <string name="network_selected">Network is selected</string>
```

Comment:
This string name is too generic (keep in mind that these names are globals for the entire app). Can you please update this to be more specific to where the string is being used?

Further, the name should not reflect its exact English contents but rather its semantic meaning (consider "all" vs. "developer_options_modify_lesson_progress_label"; the latter is more specific to the context than the actual contents of the string).

Reply:
Much better, thanks!